src/cmdhandler/command_handler.h

```diff
@@ -8,6 +8,8 @@
 #include <asio.hpp>
 #include <fstream>
 #include <stdio.h>
+#include <iostream>
+#include <dirent.h>
```

Comment:
Remove this header after switching over to filesystem for getting the available files in a directory